website/docs/enterprise/pipelines/promoting-applications.mdx

```diff
@@ -127,13 +127,33 @@ Now as soon as the `HelmRelease` on the first environment defined in the pipelin
 
 ## Pull request
 
-This section covers adding a promotion by pull request (PR) strategy, so that whenever the application defined in a pipeline is upgraded in one of the pipeline's environments, a GitHub PR is created that updates the manifest file setting the application version in the next environment.
+This section covers adding a promotion by pull request (PR) strategy, so that whenever the application defined in a pipeline
+is upgraded in one of the pipeline's environments, a PR is created that updates the manifest file setting the application version in the next environment.
 
-The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files, how to authenticate with the repository and the GitHub API, and which file hosts the version definition for each environment.
+The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files,
+how to authenticate with the repository and the git provider API, and which file hosts the version definition for each environment.
+
+### Supported Git Providers
+You could currently open pull request for the following git providers
+
+- [Github](https://github.com/)
```

Comment:
GitHub and GitLab

Reply:
sorry, could you expand on the suggestion?